mithril-aggregator/src/file_uploaders/interface.rs

```diff
@@ -0,0 +1,20 @@
+use async_trait::async_trait;
+use mithril_common::StdResult;
+use std::path::Path;
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct FileUri(pub String);
```

Comment:
Maybe a comment would be helpful for this type?

Reply:
Done ✅